packages/commonwealth/client/scripts/views/components/Header/UserDropdown/UserDropdown.tsx

```diff
@@ -84,20 +84,21 @@ const UserDropdown = () => {
         type: 'default',
         label: (
           <UserDropdownItem
-            isSignedIn={signed}
+            isSignedIn={true /*signed*/}
             hasJoinedCommunity={isActive}
             address={account.address}
           />
         ),
         onClick: async () => {
-          if (signed) {
-            return await setActiveAccount(account);
-          }
+          return await setActiveAccount(account);
+          // if (signed) {
```

Comment:
Same comment as above -- just want to make sure we don't miss it in fast follows.